### DISCUSSION TITLE
Assignment#1 about preprocessing the vocabulary

### DISCUSSION THREAD

**Discussion by localhd** [seominjoon/kaist-ai605#12] [Q&A] [answered]
Hello professor,

Maybe it is similar to the previous question on the discussion here but to clarify, I would like to ask about making the vocabulary (Problem2). 
We get tokens that are not "word" such as apostrophe, comma, or somehow blank when we only use space tokenization as guided in the assignment. 
I understood that it will not really affect the model itself, but as problem2.1, 2.2 requires the size of vocab, should we only use naive space tokenization for problem 2 to get right answer and then preprocess more if we want for the other problems?

Thanks!

**Comment by seominjoon** [comment, marked as answer]
Problem 2.1 will not require the exact number to get the full credit (in fact, there is no "exactness" in machine learning). Rather, think of it as an easy problem that will be given a full credit if you know what you are doing.

**Reply by localhd** [reply]
Thank you professor!
